app/controllers/planning_application/cil_liability_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class PlanningApplication
+  class CilLiabilityController < ApplicationController
+    include PlanningApplicationAssessable
+    before_action :set_planning_application
+    before_action :ensure_planning_application_is_validated
+
+    def edit
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def update
+      @planning_application.cil_liable = cil_liability_params[:cil_liable]
+      @planning_application.save!
```

Comment:
Could we not do the conventional

```
if @planning_application.update(cil_liability_params)
  redirect_to planning_application_assessment_tasks_path(@planning_application)
else
  render :edit
end
```

 Or we should handle the save! since it will just leave the user with a broken page

Reply:
And we'd probably want to add a success message similar to how we do in `en.yml` with

```
update:
  success:
```